tests/simd/CMakeLists.txt

```diff
@@ -15,7 +15,10 @@ message (STATUS "Configuring tests/simd")
 # ----------------------------------------------------------------------------
 
 # Search SeqAn and select dependencies.
-find_package (SeqAn REQUIRED)
+if (NOT "${SEQAN_BUILD_SYSTEM}" STREQUAL "DEVELOP")
```

Comment:
When is the SIMD test not build under `DEVELOP`

Reply:
It is built, we are just making it detect dependencies only once.